lexicons/tools/ozone/set/removeSet.json

```diff
@@ -0,0 +1,23 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.sets.removeSet",
```

Comment:
I'd go with `deleteSet` to reduce terminology conflict with `remove`.

not a blocking request, but I think it would be better.

Reply:
hmm... I kinda like removeSet since it's suffixed already, don't feel like it'll be confusing.